_rules/image-has-name-23a2a8.md

```diff
@@ -53,55 +53,64 @@ There is a known combination of a popular browser and assistive technology that
 
 #### Passed Example 1
 
-The HTML `img` element has an accessible name
+The HTML `img` element has an accessible name given by the `alt` attribute.
 
 ```html
 <img alt="W3C logo" />
 ```
 
 #### Passed Example 2
 
-The element with role of `img` has an accessible name
+The element with a [semantic role](#semantic-role) of `img` has an accessible name given by the `aria-label` attribute.
 
 ```html
 <div role="img" aria-label="W3C logo"></div>
 ```
 
 #### Passed Example 3
 
-The element has an accessible name, though the name is not always accessibility supported
+The element with a [semantic role](#semantic-role) of `img` has an accessible name given by an `aria-labelledby` attribute and element.
 
 ```html
-<img title="W3C logo" />
+<div style="display: hidden" id="img-label">W3C logo</div>
```

Comment:
It seems `display:hidden` doesn't exist.

I can't find in this documentation : https://drafts.csswg.org/css-display/#the-display-properties

But I may be wrong, I'm not a CSS expert 😇 

I suggest using either use `display:none` or `visibility:hidden` 

```html

<div style="display: none" id="img-label">W3C logo</div>